### PR TITLE
initial README and LICENSE files

### PR DESCRIPTION
**Pull request by tdyas**
Add iniital README and LICENSE files to the repository. Copied from `example-jvm` to start and then edited to be appliable to the current state of the repository. I removed the section on the test goal for now since I have not added an example of tests to this repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+# Example Goals
+
+Try these out in this repo!
+
+## Run `ktlint`
+
+```
+./pants fmt ::  # Format all files.
+./pants fmt src/jvm:  # Format only files in this directory (non-recursively).
```
[the remark this answers is not in the document]
Comment (tdyas) [reply README.md:55]: Fixed

File: README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+# Goals
+
+Pants commands are called _goals_. You can get a list of goals with
+
+```
+./pants help goals
+```
+
+Most goals take arguments to run on. To run on a single directory, use the directory name with 
+`:` at the end. To recursively run on a directory and all its subdirectories, add `::` to the
```
[the remark this answers is not in the document]
Comment (tdyas) [reply README.md:26]: Up to you. Maybe come up with whatever standard blurb you want here across the various example-* README? I am fine with whatever these say.

[the remark this answers is not in the document]
Comment (tdyas) [reply README.md:26]: And edited as suggested.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+```
+./pants dependees src/jvm/org/pantsbuild/example/json::
+./pants dependees --transitive src/jvm/org/pantsbuild/example/json/JsonExample.kt
+```
+
+## Count lines of code
+
+```
+./pants count-loc '**/*'
```
[the remark this answers is not in the document]
Comment (tdyas) [reply README.md:94]: Fixed

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Eric-Arellano) [reply README.md]: Agreed, this is fine for now

**Pull request merged**